docs/docs/how-to-deploy.mdx

```diff
@@ -69,7 +69,7 @@ for doing so, as well as manual instructions for any custom setups.
 It is also possible to deploy a Rasa assistant without Rasa X using Docker Compose. To do so, you can build your
 Rasa Assistant locally or in Docker. Then you can deploy your model in Docker Compose.
 
-* [Building a Rasa Assistant Locally](./prototype-an-assistant.mdx)
+* [Building a Rasa Assistant Locally](./playground.mdx)
```

Comment:
@erohmensing looks like you wrote most of this page - doesn't it seem a little weird to have this in here? 
User is reading about how to deploy, and we send them to the Rasa Playground to build something in the browser?

Reply:
Yeah, i was just looking at this actually. it is weird. I think the reason I did this was to say you can build locally or in docker and still deploy in docker -- but i think we can just remove this bullet point